api/v1beta1/federation_webhook.go

```diff
@@ -108,6 +110,41 @@ func (r *Federation) ValidateDelete(ctx context.Context, client client.Client, u
 	return nil
 }
 
+// validateMembers If a member is deleted, he is the initiator of the network or a member of the channel,
+// the operation is not allowed
+func validateMembers(ctx context.Context, c client.Client, oldMembers, newMembers []Member) error {
+	deleted := make(map[string]struct{})
```

Comment:
here we have a function to calculate `added`&`removed` members
https://github.com/bestchains/fabric-operator/blob/main/api/v1beta1/federation.go#L77